InternetShop/InternetShop/Controllers/UserController.cs

```diff
@@ -43,7 +43,7 @@ public async Task<IEnumerable<UserViewModel>> GetAll(CancellationToken cancellat
         }
 
         [HttpDelete("{id}")]
-        public async ValueTask Delete([FromQuery] int id, CancellationToken cancellationToken)
+        public async ValueTask Delete(int id, CancellationToken cancellationToken)
```

Comment:
it shouldn't be here. Because it does not apply to this PR

Reply:
done

Reply:
done